portfolio/src/main/webapp/index.html

```diff
@@ -5,26 +5,243 @@
+                in Religion and Psychology. I am mostly interested in Artificial Intelligence as well as Computer Systems.
+                Aside of classes, I work as a Teaching Assistant for Data Structures and Algorithms and also
+                as a Tour Guide for the Admissions and Financial Aid Office.<br><br>
+                In my free time, you will either find me playing piano or playing video games. I am also a big fan of 
+                outdoor activities such as hiking and kayaking. Therefore, if you do not find me in the music room, or in my room,
+                then you might be able to see me on Genesee River, trying to prevent myself from drowning with my kayak :)
+            </p>
+            <p id="string-container">Click here to see stuff</p>
+            <button onclick="showServerString();">
+                    string
+            </button>
+        </div>
+        <div class="w50">
+            <img src="images/resizedimg.jpg" class="img">
+        </div>
+        </div>
+      </section>
+      <script type="text/javascript">
+        window.addEventListener('scroll', function()
+        {
+            var header = document.querySelector('header');
+            header.classList.toggle('sticky', window.scrollY > 0);
+
+        });
+      </script>  
+    <section class="skills">
+        <div class="heading">
+            <h2>Skills and Tools</h2>
+        </div>
+        <p>Here are the languages, frameworks, and tools I have experience with</p>
+        <div class="tools">
+            <ul>
+              <li>JAVA</li>
+              <li>C</a></li>
+              <li>Python</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>MySQL</li>
+              <li>Lua</li>
+            </ul>
+        </div>
+        <div class="courses">
+            <div class="heading">
+                <h2>Technical Courses</h2>
+            </div>
+            <ul>
+                <li>Introduction to Computer Science</li>
+                <li>Data Structures and Algorithms</li>
+                <li>Discrete Maths</li>
+                <li>Linear Algebra and Differential Eqns</li>
+                <li>Formal Models of Computation</li>
+                <li>Introduction to Artificial Intelligence</li>
+                <li>Computer Organization</li>
+            </ul>
+        </div>
+        <div class="experience">
+            <div class="heading">
+                <h2>Experience</h2>
+            </div>
+            <div class="content">
+                <div class="one-grid">Teaching Assistant @UofR</div>
+                    <div class="time-span">
+                        Lorem ipsum dolor sit amet
+                    </div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent feugiat ac nisi a sagittis. Vivamus luctus id quam sit amet fringilla. Nam auctor libero sed iaculis posuere. Donec nec purus et urna efficitur convallis. 
```

Comment:
I'm assuming this placeholder text is just here for layout and will be replaced later?